libbpf-rs/src/object.rs

```diff
@@ -334,6 +353,58 @@ impl Object {
         self.map(name).unwrap().unwrap()
     }
 
+
+    pub fn get_maps(&mut self) -> Result<&HashMap<String, Map>> {
```

Comment:
Also this should probably be named `maps(..)` and not `get_maps(..)`. See https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter